uninstaller/pre-delete.py

```diff
@@ -32,40 +32,126 @@
 
 
 def check_pfserver_status(service_name, namespace, port, user, password):
-    host = "%s.%s.svc.cluster.local" % (service_name, namespace)
-    print(host)
-    
+    print("check_pfserver_status service_name=[%s] namespace=[%s] port=[%s] user=[%s] password=[%s]" % (
+    service_name, namespace, port, user, password))
+    host = service_name
+
     client = paddleflow.Client(host, user, password, port)
     ret, response = client.login(user, password)
     if not ret:
         print(response)
-        err_msg = "client login failed, with host[%s], user[%s], password[%s], port[%s], response[%s]" % (host, user, password, port, response)
+        err_msg = "client login failed, with host[%s], user[%s], password[%s], port[%s], response[%s]" % (
+        host, user, password, port, response)
         raise Exception(err_msg)
-    
+    sum = 1
+    while sum != 0:
+        ppl_num = clean_pipelines(client)
+        job_num = clean_jobs(client)
+        sum = ppl_num + job_num
+        if sum == 0:
+            # clean queue only if no running ppl or job
+            sum = clean_default_queue(client)
+        clean_storage(client)
+        if sum != 0:
+            print("\nscheduling clean paddleflow resource, current resource is %d" % sum)
+            time.sleep(5)
+
+
+def clean_pipelines(client):
+    print("clean_pipelines starting")
     active_status_list = ["initiating", "pending", "running", "terminating"]
-    while True:
-        ret, response = client.list_run(status=".".join(active_status_list))
-        if not ret:
-            print(response)
-            err_msg = "list run failed, with response[%s]" % (response)
-            raise Exception(err_msg)
-        
-        run_list = response["runList"]
-        if len(run_list) != 0:
-            err_msg = "there are [%s] active runs" % len(run_list)
-            print(err_msg)
-            time.sleep(5) 
-        else:
-            err_msg = "no active run, quit pre-delete check"
-            print(err_msg)
-            return 0
+    ret, response = client.list_run(status=".".join(active_status_list))
+    if not ret:
+        print(response)
+        err_msg = "list run failed, with response[%s]" % (response)
+        raise Exception(err_msg)
+
+    run_list = response["runList"]
+    if len(run_list) != 0:
+        err_msg = "there are [%s] active runs" % len(run_list)
+        print(err_msg)
+    else:
+        err_msg = "no active run, quit clean_pipelines check"
+        print(err_msg)
+    print("clean_pipelines end\n\n")
+    return len(run_list)
+
+
+def clean_storage(client):
+    print("clean_storage")
+    print("clean_storage end\n\n")
+    pass
+
+
+def clean_default_queue(client):
+    print("clean_queue")
```

Comment:
In CCE environment, PaddleFlow creates  queue with name `default-queue`, and we clean default-queue only. Why not use `client.show_queue` to query default-queue?